pom.xml

```diff
@@ -28,15 +28,15 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.50</version>
+        <version>3.54</version>
         <relativePath />
     </parent>
     <groupId>org.jenkins-ci.plugins.workflow</groupId>
     <artifactId>workflow-cps-global-lib</artifactId>
     <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <name>Pipeline: Shared Groovy Libraries</name>
-    <url>https://wiki.jenkins.io/display/JENKINS/Pipeline+Shared+Groovy+Libraries+Plugin</url>
+    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
```

Comment:
This should be paired with a patch to `README.md` importing the wiki content using https://jenkins-wiki-exporter.jenkins.io/

Reply:
I have no interest in doing that, so I will just revert this change and let the plugin maintainer take care of it.